blog/_posts/2021-07-19-scala-3.0.2RC1-is-here.md

```diff
@@ -0,0 +1,224 @@
+---
+layout: blog-detail
+post-type: blog
+by: Michał Pałka, VirtusLab
+title: Scala 3.0.1 and 3.0.2-RC1 are here!
+---
+
+Greetings from the Scala 3 team! We are glad to announce that Scala 3.0.1 and 3.0.2-RC1 are now officially out.
+As no critical bugs have been found in the previously released Scala 3.0.1-RC2, it has been promoted to 3.0.1, which is the first stable release after 3.0.0. Scala 3.0.2-RC1, in turn, incorporates new language improvements and bug fixes. You can expect the release of stable 3.0.2 and a release candidate for a the next version in 6 weeks from now (1st August).
+
+# Improved insertion of semicolons in logical conditions
```

Comment:
So all these changes are only for .2-RC1? It's really confusing because you're talking about .1 as well. Could we at least add a header clarifying that these release notes are only for 3.0.2-RC1

IMO it would be much clearer, and much more appropriate for users' use-cases, to:
1. split this into two blog posts: 3.0.1 and 3.0.2-RC2. I know it's more convenient to write but it's confusing to read.
2. In the 3.0.1 blogpost list *all* the changes since 3.0.0. At the moment I need to manually read the RC1 and RC2 notes like it's a story. I'd say the vast majority of users aren't so invested and just want to know what's changed.

Reply:
IMO realistically we only need the 3.0.1 post, disregarding the RC1 changes altogether. The status of RCs is not the same as before 3.0.0. Before 3.0.0, the RCs were the de-facto main releases. But now, they are mere precursors to stable releases. The community should focus the ecosystem on the stable releases, not RCs.

So it would make sense to downplay the role of RCs and not mention their changes in the article if it makes it confusing to read. We can just mention that RC was released in the header, refer the interested users to the changelog on GitHub, and then talk only about the stable release changes.

Reply:
Yeah, we had some discussions about this and in the future we're going to focus on stable releases. For now however we're in some some transition state between the old and the new release process and for the blogpost this is still going the old way. I tried to make it explicit now that the described changes are for 3.0.2-RC1